js/src/product-feed/review-request/review-request-modal.js

```diff
@@ -43,6 +46,32 @@ const ReviewRequestModal = ( {
 		} );
 	};
 
+	const handleReviewRequest = () => {
+		if ( isRequestingReview ) return;
+
+		setIsRequestingReview( true );
+		recordEvent( 'gla_request_review' );
+
+		mcRequestReview()
+			.then( () => {
+				createNotice(
+					'success',
+					__(
+						'Your account review was successfully requested.',
+						'google-listings-and-ads'
+					)
+				);
+				setIsRequestingReview( false );
+				onClose( 'request-review-success' );
+				recordEvent( 'gla_request_review_success' );
+			} )
+			.catch( ( error ) => {
+				recordEvent( 'gla_request_review_failure', {
+					error: error?.message,
```

Comment:
❓ For the purpose of event tracking, do we need to record the error message? I'm a bit worried about whether there would be sensitive information in the error message.